examples/jsm/physics/RapierPhysics.js

```diff
@@ -28,6 +28,17 @@ function getShape( geometry ) {
 		const radius = parameters.radius !== undefined ? parameters.radius : 1;
 		return RAPIER.ColliderDesc.ball( radius );
 
+	} else if ( geometry.type === 'BufferGeometry' ) {
+
+		const vertices = geometry.getAttribute( 'position' ).array;
```

Comment:
This code is not safe if data are interleaved.

You have to iterate over the position attribute, extract the vertex and then add it to a new array.
```js
const vertices = [];
const vertex = new Vector3();

for ( let i = 0, il = position.count; i < il; i ++ ) {

    vertex.fromBufferAttribute( position, i );
    vertices.push( vertex.x, vertex.y, vertex.z );

}
```
Index data are normally not interleaved so the rest of the code should be fine.

Reply:
You are right! I've already made the changes and tested with an interleaved buffer geometry. Thanks for the tip!